horse/recommendation/service.py

```diff
@@ -0,0 +1,6 @@
+import abc
+
+
+class RecommendationService(metaclass=abc.ABCMeta):
+    def recommend(self, user):
+        raise NotImplemented
```

Comment:
good to know, i didn't have a chance to use 'real' abstract classes in Python :)